src/main/java/appeng/integration/abstraction/IRC.java

```diff
@@ -21,9 +21,16 @@
 
 import net.minecraft.item.ItemStack;
 
+import appeng.integration.IIntegrationModule;
 
-public interface IRC
+
+public interface IRC extends IIntegrationModule
 {
 
-	void rockCrusher( ItemStack input, ItemStack output );
+	default void rockCrusher( ItemStack input, ItemStack output ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)